src/Administration/Login/Config/TemplateData.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Administration\Login\Config;
+
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * @internal
+ */
+#[Package('after-sales')]
+final class TemplateData implements \JsonSerializable
+{
+    public function __construct(
+        public readonly string $random,
+        public readonly bool $show,
```

Comment:
This flag also seems to be not needed anymore. I guess just checking if the `$url` isn't empty should be enough to decide if we should display the button or not